integreat_cms/cms/models/pois/poi.py

```diff
@@ -93,6 +94,20 @@ class POI(AbstractContentModel):
         default=get_default_opening_hours,
         verbose_name=_("opening hours"),
     )
+    organization = models.ForeignKey(
+        Organization,
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="pois",
+        verbose_name=_("organization"),
```

Comment:
```suggestion
        verbose_name=_("organization"),
        help_text=_("Specify which organization operates this location."),
```
Maybe add a help text for this field as well?